yt/utilities/command_line.py

```diff
@@ -754,21 +751,18 @@ def __call__(self, args):
             p = SlicePlot(ds, args.axis, args.field)
         from yt.visualization.mapserver.pannable_map import PannableMapServer
         PannableMapServer(p.data_source, args.field)
-        import yt.extern.bottle as bottle
+        import bottle
```

Comment:
Can you wrap this in a `try, except ImortError block` suggesting to install bottle of importing it doesn’t work?

Reply:
Something like:
```
        try:
            import bottle
        except ImportError:
            raise RuntimeError(
                "The mapserver requires the package `bottle' to be "
                "installed. Please install using `pip install bottle`."
            )
```

Reply:
yup